core/src/test/java/org/infinispan/configuration/serializer/AbstractConfigurationSerializerTest.java

```diff
@@ -0,0 +1,96 @@
+package org.infinispan.configuration.serializer;
+
+import org.infinispan.commons.configuration.attributes.Attribute;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.commons.util.FileLookupFactory;
+import org.infinispan.configuration.cache.AbstractStoreConfiguration;
+import org.infinispan.configuration.cache.ClusterLoaderConfiguration;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.SingleFileStoreConfiguration;
+import org.infinispan.configuration.cache.StoreConfiguration;
+import org.infinispan.configuration.parsing.ConfigurationBuilderHolder;
+import org.infinispan.configuration.parsing.ParserRegistry;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public abstract class AbstractConfigurationSerializerTest {
```

Comment:
I need the annotation ```@Test(groups = "functional")``` otherwise I can't run the test via command line. If I try
```
t -pl persistence/jdbc/ -Dtest=persistence.jdbc.configuration.ConfigurationSerializerTest
```
it would fail without finding the test.